x-pack/platform/plugins/shared/inference/server/tracing/util/flatten_attributes.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AttributeValue } from '@opentelemetry/api';
+import { isArray, isPlainObject } from 'lodash';
+
+export function flattenAttributes(
```

Comment:
I'm assuming we are not using `flattenObject` from '@kbn/object-utils' here cause of the type `AttributeValue`?

Reply:
Yeah that one doesn't flatten arrays.